### DISCUSSION TITLE
Simon io 100 switch slow response

### DISCUSSION THREAD

**Discussion by davidcalleja** [zwave-js/zwave-js-ui#3402] [Q&A]
I'm using a simon io 100 switch (10002034-13X) to switch on or off a Hue light (always connected to grid) but each click takes 2 seconds to be captured by `zwave-js-ui` but if you send a switch action from `zwave-js-ui` to device is instantaneously. 

Anyone have idea about this problem?

**Comment by robertsLando** [comment]
CC @AlCalzone
